pom.xml

```diff
@@ -459,6 +460,25 @@
             <version>${micrometer.version}</version>
         </dependency>
 
+        <!-- netcdf4 -->
+
+        <dependency>
+            <groupId>edu.ucar</groupId>
+            <artifactId>netcdf4</artifactId>
+            <version>${netcdf4.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Haven't we included this elsewhere in the pom.xml?

Reply:
@kaeline this one is for you

Reply:
No, I don't think this dependency is duplicated.